Bindings/bindings.json

```diff
@@ -113,6 +113,26 @@
                     "label": "$eventHubTrigger_connection_label",
                     "help": "$eventHubTrigger_connection_help",
                     "placeholder": "[variables('selectConnection')]"
+                },
+                {
+                    "name": "cardinality",
+                    "value": "enum",
+                    "resource": "EventHub",
+                    "required": false,
+                    "defaultValue": "many",
+                    "enum": [
+                        {
+                            "value": "one",
+                            "display": "one"
+                        },
+                        {
+                            "value": "many",
+                            "display": "many"
+                        }
+                    ],
+                    "label": "$eventHubTrigger_cardinality_label",
```

Comment:
Add corresponding values for $eventHubTrigger_cardinality_label and  $eventHubTrigger_cardinality_help in resource files.
Here is a [sample ](https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Resources/Resources.resx#L129).